web-registry/src/pages/Dashboard/MyBridge/MyBridge.tsx

```diff
@@ -1,5 +1,39 @@
+import { useCallback, useMemo } from 'react';
+import { Outlet } from 'react-router-dom';
+
+import { IconTabProps } from 'web-components/lib/components/tabs/IconTab';
+
+import { useLedger } from 'ledger';
+
 import { Bridge } from 'components/organisms';
 
 export const MyBridge = (): JSX.Element => {
-  return <Bridge privateAccess />;
+  const { wallet } = useLedger();
+
+  const MyBridgeOutlet = useCallback(
+    (): JSX.Element => (
+      <Outlet
+        context={{ accountAddress: wallet?.address, privateAccess: true }}
+      />
+    ),
+    [wallet?.address],
+  );
+
+  const tabs: IconTabProps[] = useMemo(
+    () => [
+      {
+        label: 'Bridgable ecocredits',
+        href: '/ecocredits/bridge/bridgable',
+        content: <MyBridgeOutlet />,
```

Comment:
In order to leverage the <TabPanel> styles from IconTabs, we have to send in the Outlet as `content`.

An alternative would be adding an optional `outlet` prop to IconTabs, but this felt odd.